Frontend/src/pages/registration_edit/components/Refunds.jsx

```diff
@@ -42,7 +46,7 @@ export default function Refunds({ open, onExit, userId, competitionId }) {
         <Modal.Content>
           <Table>
             <Table.Header>
-              <Table.Header> Amount </Table.Header>
+              <Table.Header>{t('')}</Table.Header>
```

Comment:
This is empty.

Reply:
thanks. As this is organizer only, this won't translated